db/migrate/20150907120454_create_container_volumes.rb

```diff
@@ -0,0 +1,50 @@
+class CreateContainerVolumes < ActiveRecord::Migration
+  def up
+    create_table :container_volumes do |t|
+      t.belongs_to :container_group, :type => :bigint
+      t.string     :type
+      t.string     :name
+      t.string     :host_path
+      t.string     :medium_type
+      t.string     :pd_name
+      t.string     :gce_fs_type
+      t.integer    :gce_partition
+      t.boolean    :gce_read_only
+      t.string     :aws_volume_id
+      t.string     :aws_fs_type
+      t.string     :aws_partition
+      t.boolean    :aws_read_only
+      t.string     :git_repository
+      t.string     :git_revision
+      t.string     :secret
+      t.string     :nfs_server
+      t.string     :nfs_path
+      t.boolean    :nfs_read_only
+      t.string     :target_portal
+      t.string     :iqn
+      t.integer    :lun
```

Comment:
if `iqn`, `lun` and `lun` are related to `iscsi` they should have the `iscsi_` prefix I susppose.